src/components/organisms/PublicFooter.vue

```diff
@@ -0,0 +1,76 @@
+<script setup>
+import { computed } from 'vue'
+import { APP_TITLE } from '~/constants'
+import DcLocalePicker from '../molecules/DcLocalePicker.vue';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n()
+
+const menu = computed(() => [
+  {
+    title: t('button.about'),
+    href: '/#about',
```

Comment:
Why do we need to put a # before the route name?

Reply:
I was testing this to send the user to the home page `/` at the anchor `#about`, instead of sending the user to the page `"/about". I gonna remove it.